rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc

```diff
@@ -0,0 +1,60 @@
+= RPA Bot 1.6.2 Release Notes
+
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== March 25, 2024
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|The custom message created from the *Write Bot Message* action step and now displays the full 2,000 characters and no longer truncates the message after 1,000 characters.| W-15252227 
+|RPA Bot now removes sessions that are in a state of "Disconnected" to prevent RPA Bot from attempting to connect to and create Secure Sessions from "Disconnected" sessions.| W-15241413
+|===
+
+== February 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The System Settings panel now has a `Bot Service State` entry, which shows the current Windows Service state and configuration.
+* The *Password* and *Current User* fields on the Session Settings panel are renamed *Default Session User Password* and *RPA Bot Configurator User* respectively.
+* A warning message has been added to indicate that if a deployed process is running on the bot instance, the process may be canceled if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
+* When a change is made to the RPA Bot Configurator while a process is running, you can now select from multiple restart, exit, and cancel options to decide how you want to proceed with the process that is running and the changes to the RPA Bot Configurator.
```

Comment:
```suggestion
* When a change is made to the RPA Bot Configurator while a process is running, you can now select from multiple restart, exit, and cancel options to decide how to continue with the process that is running and the changes to the RPA Bot Configurator.
```

Reply:
Is the use of "the" consistent? Docs names "RPA Bot" without "the", which seems to be OK as this is a proper name. But docs use "the" with "RPA Bot Configurator", which seems to be a proper name as well as it is capitalized.